skimage/measure/simple_metrics.py

```diff
@@ -142,3 +145,48 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def enhancement_measure(image: np.ndarray,
+                        eps: float = 1e-6,
+                        size: int = 3) -> float:
+    """ The image enhancement measure called EME based on [1]_.
```

Comment:
Add one sentence to explain what EME is measuring, like "EME is a measure of average local contrast, measured on a logarithmic scale inside windows of user-provided size".

Reply:
I added more explanations to the doc files, thanks for pointing that out.